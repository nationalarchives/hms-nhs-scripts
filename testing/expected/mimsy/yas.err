Error: illegal fraction ".1" in "0.1;0.2" ("years at sea") at line 4 of input/mimsy/yas.csv.
       Fraction must be one of: .08, .17, .25, .33, .42, .5, .58, .67, .75, .83, .92.
Error: illegal fraction ".2" in "0.1;0.2" ("years at sea") at line 4 of input/mimsy/yas.csv.
       Fraction must be one of: .08, .17, .25, .33, .42, .5, .58, .67, .75, .83, .92.
Error: too many "years at sea" sub-fields in "0;1;2" at line 5 of input/mimsy/yas.csv.
Error: bad number format "a" in "a;b" ("years at sea") at line 6 of input/mimsy/yas.csv.
Error: bad number format "b" in "a;b" ("years at sea") at line 6 of input/mimsy/yas.csv.
Error: "years at sea" value for "merchant" or "navy" missing in "0" at line 7 of input/mimsy/yas.csv.
Error: bad number format "" in ";" ("years at sea") at line 8 of input/mimsy/yas.csv.
Error: bad number format "" in ";" ("years at sea") at line 8 of input/mimsy/yas.csv.
Error: bad number format "0.1.2" in "0.1.2;2.1.0" ("years at sea") at line 9 of input/mimsy/yas.csv.
Error: bad number format "2.1.0" in "0.1.2;2.1.0" ("years at sea") at line 9 of input/mimsy/yas.csv.
Warning: blank in "years at sea" at row 10 of input/mimsy/yas.csv (volume 2, page 10)
Error: "years at sea" value for "merchant" or "navy" missing in "" at line 10 of input/mimsy/yas.csv.
Error: bad number format "" in "0;" ("years at sea") at line 11 of input/mimsy/yas.csv.
Error: bad number format "" in "; 0" ("years at sea") at line 12 of input/mimsy/yas.csv.
