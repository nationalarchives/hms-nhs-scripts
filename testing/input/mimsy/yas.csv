volume,page,admission number,date of entry,name,quality,age,place of birth,port sailed out of,years at sea,last services,under what circumstances admitted (or nature of complaint),date of discharge,how disposed of,number of days victualled
2,2,1,Feb 02 1822,Bob,Boy,15,South Mimms,Dover,00;00,Nonesuch,Sleepy,Feb 03 1822,Splashed,1
2,3,1,Feb 02 1822,Bob,Boy,15,South Mimms,Dover,10.50;09.5,Nonesuch,Sleepy,Feb 03 1822,Splashed,1
2,4,1,Feb 02 1822,Bob,Boy,15,South Mimms,Dover,0.1;0.2,Nonesuch,Sleepy,Feb 03 1822,Splashed,1
2,5,1,Feb 02 1822,Bob,Boy,15,South Mimms,Dover,0;1;2,Nonesuch,Sleepy,Feb 03 1822,Splashed,1
2,6,1,Feb 02 1822,Bob,Boy,15,South Mimms,Dover,a;b,Nonesuch,Sleepy,Feb 03 1822,Splashed,1
2,7,1,Feb 02 1822,Bob,Boy,15,South Mimms,Dover,0,Nonesuch,Sleepy,Feb 03 1822,Splashed,1
2,8,1,Feb 02 1822,Bob,Boy,15,South Mimms,Dover,; ,Nonesuch,Sleepy,Feb 03 1822,Splashed,1
2,9,1,Feb 02 1822,Bob,Boy,15,South Mimms,Dover,0.1.2;2.1.0,Nonesuch,Sleepy,Feb 03 1822,Splashed,1
2,10,1,Feb 02 1822,Bob,Boy,15,South Mimms,Dover,,Nonesuch,Sleepy,Feb 03 1822,Splashed,1
2,11,1,Feb 02 1822,Bob,Boy,15,South Mimms,Dover,0; ,Nonesuch,Sleepy,Feb 03 1822,Splashed,1
2,12,1,Feb 02 1822,Bob,Boy,15,South Mimms,Dover,; 0,Nonesuch,Sleepy,Feb 03 1822,Splashed,1
