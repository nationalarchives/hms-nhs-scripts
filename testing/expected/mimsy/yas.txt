DSH/2/2,admission number, 1; date of entry, 02-02-1822; name, Bob; quality, Boy; age, 15; place of birth, South Mimms; port sailed out of, Dover; years at sea, 0: 0; last services, Nonesuch; under what circumstances admitted (or nature of complaint), Sleepy; date of discharge, 03-02-1822; how disposed of, Splashed; number of days victualled, 1;
DSH/2/3,admission number, 1; date of entry, 02-02-1822; name, Bob; quality, Boy; age, 15; place of birth, South Mimms; port sailed out of, Dover; years at sea, 10.5: 9.5; last services, Nonesuch; under what circumstances admitted (or nature of complaint), Sleepy; date of discharge, 03-02-1822; how disposed of, Splashed; number of days victualled, 1;
DSH/2/4,admission number, 1; date of entry, 02-02-1822; name, Bob; quality, Boy; age, 15; place of birth, South Mimms; port sailed out of, Dover; years at sea, 0.1: 0.2; last services, Nonesuch; under what circumstances admitted (or nature of complaint), Sleepy; date of discharge, 03-02-1822; how disposed of, Splashed; number of days victualled, 1;
DSH/2/5,admission number, 1; date of entry, 02-02-1822; name, Bob; quality, Boy; age, 15; place of birth, South Mimms; port sailed out of, Dover; years at sea, 0:1:2; last services, Nonesuch; under what circumstances admitted (or nature of complaint), Sleepy; date of discharge, 03-02-1822; how disposed of, Splashed; number of days victualled, 1;
DSH/2/6,admission number, 1; date of entry, 02-02-1822; name, Bob; quality, Boy; age, 15; place of birth, South Mimms; port sailed out of, Dover; years at sea, a: b; last services, Nonesuch; under what circumstances admitted (or nature of complaint), Sleepy; date of discharge, 03-02-1822; how disposed of, Splashed; number of days victualled, 1;
DSH/2/7,admission number, 1; date of entry, 02-02-1822; name, Bob; quality, Boy; age, 15; place of birth, South Mimms; port sailed out of, Dover; years at sea, 0; last services, Nonesuch; under what circumstances admitted (or nature of complaint), Sleepy; date of discharge, 03-02-1822; how disposed of, Splashed; number of days victualled, 1;
DSH/2/8,admission number, 1; date of entry, 02-02-1822; name, Bob; quality, Boy; age, 15; place of birth, South Mimms; port sailed out of, Dover; years at sea, : ; last services, Nonesuch; under what circumstances admitted (or nature of complaint), Sleepy; date of discharge, 03-02-1822; how disposed of, Splashed; number of days victualled, 1;
DSH/2/9,admission number, 1; date of entry, 02-02-1822; name, Bob; quality, Boy; age, 15; place of birth, South Mimms; port sailed out of, Dover; years at sea, 0.1.2: 2.1.0; last services, Nonesuch; under what circumstances admitted (or nature of complaint), Sleepy; date of discharge, 03-02-1822; how disposed of, Splashed; number of days victualled, 1;
DSH/2/10,admission number, 1; date of entry, 02-02-1822; name, Bob; quality, Boy; age, 15; place of birth, South Mimms; port sailed out of, Dover; years at sea, ; last services, Nonesuch; under what circumstances admitted (or nature of complaint), Sleepy; date of discharge, 03-02-1822; how disposed of, Splashed; number of days victualled, 1;
DSH/2/11,admission number, 1; date of entry, 02-02-1822; name, Bob; quality, Boy; age, 15; place of birth, South Mimms; port sailed out of, Dover; years at sea, 0: ; last services, Nonesuch; under what circumstances admitted (or nature of complaint), Sleepy; date of discharge, 03-02-1822; how disposed of, Splashed; number of days victualled, 1;
DSH/2/12,admission number, 1; date of entry, 02-02-1822; name, Bob; quality, Boy; age, 15; place of birth, South Mimms; port sailed out of, Dover; years at sea, : 0; last services, Nonesuch; under what circumstances admitted (or nature of complaint), Sleepy; date of discharge, 03-02-1822; how disposed of, Splashed; number of days victualled, 1;
